docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.5'
+services:
+  dynamodb-local:
```

Comment:
if we don't need this service right now, will you coment this service out until ddb backend is done?

Reply:
Agreed.